tox.ini

```diff
@@ -30,8 +30,11 @@ deps =
 commands =
     {envpython} --version
     trial --version
-    coverage run --source klein --omit klein/test/* --branch {envdir}/bin/trial {posargs:klein}
+    coverage erase
```

Comment:
Well, the problem is, to make the paths look correct, you need to do 'coverage combine' :(